lib/Doctrine/Migrations/Tools/Console/Command/DumpSchemaCommand.php

```diff
@@ -152,4 +149,14 @@ public function execute(
 
         return 0;
     }
+
+    private function checkNotTooManyMigrations(string $namespace) : void
```

Comment:
I think this might be more accurate:
```suggestion
    private function checkNoPreviousDumpExists(string $namespace) : void
```